src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -20,9 +27,13 @@
 	{#if back}
 		<Back />
 	{:else}
-		<div class="flex p-4 items-center">
+		<button
```

Comment:
Can we rather use `a` or is there is some issues that prevent us to use a link?

Reply:
ah right! i forgot that it is preferred: have a look